actions/recipe_action.go

```diff
@@ -6,7 +6,6 @@ Include a recipe.
 Yaml syntax:
  - action: recipe
    recipe: path to recipe
-   inherit: bool
```

Comment:
Squash this into your first commit please

Reply:
Squashed